Source/Core/Core/PrimeHack/PrimeUtils.h

```diff
@@ -41,6 +43,17 @@ void set_beam_owned(int index, bool owned);
 void set_visor_owned(int index, bool owned);
 void set_cursor_pos(float x, float y);
 
+
+std::string as_hex_string(u32 val);
```

Comment:
It may be worth changing this from only supporting hex to taking in any format like DevInfo.